ocs_ci/helpers/dr_helpers.py

```diff
@@ -535,3 +536,63 @@ def wait_for_all_resources_deletion(
         pvc_obj.ocp.wait_for_delete(
             resource_name=pvc_obj.name, timeout=timeout, sleep=5
         )
+
+
+def resources_cleanup(namespace):
```

Comment:
@sidhant-agrawal , may be we can rename this function specific to resources_cleanup_rbd or something, because when cephfs comes up we need another method as well I guess.

Reply:
I believe same function can be used with some minor modification for cephfs. For now, we can keep as it is.

Reply:
Made few changes to handle cephfs in the same function.